odoo/addons/base/models/ir_translation.py

```diff
@@ -564,7 +564,10 @@ def write(self, vals):
     def unlink(self):
         self.check('unlink')
         self._modified()
-        return super(IrTranslation, self.sudo()).unlink()
+
+        # avoid calling super().unlink(), as it clears the whole cache
+        # This method may be called when updating translated fields, clearing the whole cache is overkill.
+        self._cr.execute("DELETE FROM ir_translation WHERE id=ANY(%s)", [list(self.ids)])
```

Comment:
`self.ids` is always a list
```suggestion
        self._cr.execute("DELETE FROM ir_translation WHERE id=ANY(%s)", [self.ids])
```